library.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "ArduinoSort",
+  "keywords": "sort, sorting",
+  "description": "Easy sorting of arrays for Arduino, with focus on low memory footprint.",
+  "homepage": "https://github.com/emilv/ArduinoSort",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/emilv/ArduinoSort.git"
+  },
+  "version": "v1.0-rc.1",
+  "authors": {
```

Comment:
Please remove `authors` and `examples` blocks, PlatformIO calculates them automatically.

Reply:
The library.properties is just another format? Is it needed?

Reply:
PlatformIO supports both formats. `library.json` is officially supported.